### PR TITLE
Build Tensile PR in rocBLAS 

### PR DESCRIPTION
**Pull request by amdkila**
- This PR builds Tensile PRs in rocBLAS. This can be viewed here: http://10.216.151.18:8080/job/Dependency%20Projects/job/Tensile/view/change-requests/ 
- Passed CI

### CODE REVIEW

File: .jenkins/Jenkinsfile2 [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env groovy
```
Comment (sdquiring) [inline .jenkins/Jenkinsfile2]: This file should have a more descriptive name.  Do we want to move the other jenkinsfile into here as well?

Comment (amdkila) [reply .jenkins/Jenkinsfile2]: Yeah, definitely agree about the name. Not sure what to call it, definitely open to suggestions. As for moving the other jenkinsfile here, sure. I've added .jenkins/jenkins to RocmSoftwarePlatform as an acceptable Jenkins file location, so I can move the current Jenkins file.

Comment (sdquiring) [reply .jenkins/Jenkinsfile2]: The main thing would be to put 'rocblas' somewhere in the name so that it's obvious that this is building that project.

**Comment by amdkila** [comment]
Compile stage failures appear to be legitimate errors. This should be safe to merge.

File: docker/dockerfile-build-ubuntu-rock [hunk 2]
```diff
@@ -4,28 +4,34 @@
 ARG base_image
 
 FROM ${base_image}
-LABEL maintainer="saad.rahim@amd"
+LABEL maintainer="tensile.maintainer@amd.com"
```
Comment (saadrahim) [inline docker/dockerfile-build-ubuntu-rock]: check other projects for convention

**Pull request closed without merging**